browser-test/src/applicant/northstar_application_version_fast_forward.test.ts

```diff
@@ -851,7 +851,7 @@ test.describe(
             ])
           })
 
-          return await test.step('add questions to program and republish program', async () => {
+          return await test.step('add questions to program and republish disabled program', async () => {
```

Comment:
By only looking at the test output it took me some time to understand why the next step was redirecting to the disabled url. Just making it more clear